README.rst

```diff
@@ -94,8 +94,12 @@ Raw HTML    ``extra.html('<div>Additional HTML</div>')``
 Plain text  ``extra.text('Add some simple Text')``
 URL         ``extra.url('http://www.example.com/')``
 Image       ``extra.image(image, mime_type='image/gif', extension='gif')``
+Image       ``extra.image('file:///path/to/file.png')``
+Image       ``extra.image('http://some_image.png')``
 ==========  ============================================
 
+**Note**: When using ``--self-contained-html``, images added as files or links may not work as expected.
```

Comment:
I also think we should move this to the 'Creating a self-contained report' section, and reference/link to that section here.